programs/address-lookup-table/src/instruction.rs

```diff
@@ -47,6 +47,14 @@ pub enum ProgramInstruction {
     ///   3. `[]` System program for CPI.
     ExtendLookupTable { new_addresses: Vec<Pubkey> },
 
+    /// Deactivate an address lookup table, making it unusable and
+    /// eligible for closure after a short period of time.
+    ///
+    /// # Account references
+    ///   0. `[WRITE]` Address lookup table account to deactivate
+    ///   1. `[SIGNER]` Current authority
+    DeactivateLookupTable,
```

Comment:
Just double-checking, this isn't being used yet, correct? If so, this reordering of instructions will break existing clients

Reply:
Correct!